docker/multi-process/README.md

```diff
@@ -75,9 +76,10 @@ To link to another mysql container, for example:
 
 To link to another container named 'postgres':
 
-    docker run --name huginn_postgres \
+    docker run --rm --name huginn_postgres \
```

Comment:
Not sure if we should add that, an unexperienced docker user might not notice the `--rm` flag and lose the database after stopping the container.

Reply:
I added this just to be consistent with the `huginn_mysql` example. But I agree, perhaps we should remove this flag for all database container examples.

Reply:
I see, thanks!